src/sql/sqlops.proposed.d.ts

```diff
@@ -587,7 +587,8 @@ declare module 'sqlops' {
 		string = 'string',
 		category = 'category',
 		boolean = 'boolean',
-		editableCategory = 'editableCategory'
+		editableCategory = 'editableCategory',
```

Comment:
I think we shouldn't need to update the sqlops declaration file. My understanding is typescript can merge enum's fine as long as they don't have conflicting values (string = string and editableCategory = string).

Reply:
So it looks like I need to keep the component addition in. In sqlExtHost.api.impl.ts we're mapping the sqlExtHostTypes enum to the namespaces and that's failing for sqlops since we don't have it defined. 

Is that what you were referring to? I'm not sure based on your comment - I didn't modify the other definitions at all.

Reply:
makes sense, maybe we should have a separate exthosttypes for sqlops than azdata to avoid this and making changes to the types that sqlops is using.

Reply:
Yeah, I was thinking something like that. But we're going to be getting rid of sqlops within the next release or two so I think it's fine to just live with this for now